cmd/kured/main.go

```diff
@@ -209,7 +209,7 @@ func rebootBlockingPodsExist(client *kubernetes.Clientset, nodeID string) bool {
 func releaseBlockingPodsNotready(client *kubernetes.Clientset, nodeID string) bool {
 	fieldSelector := fmt.Sprintf("spec.nodeName=%s", nodeID)
 	for _, labelSelector := range releasePodSelectors {
-		podList, err := client.CoreV1().Pods("").List(metav1.ListOptions{
+		podList, err := client.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{
```

Comment:
Are you sure context.TODO is appropriate here?
Normally a context object is passed through all layers, so state can propagate. TODO is mostly a placeholder.
Do you intend to make releaseBlockingPodsNotready accept a context argument?

Reply:
They do it [here](https://github.com/swisstxt/kured/blob/ddef56b67478fb7d032a59fd8b9dcfcf3af25a11/cmd/kured/main.go#L184), so I thought I do it the same here too. :joy: 

Actually I have no plan what I'm doing, but as far as I can tell it works 
![](https://media4.giphy.com/media/SRx5tBBrTQOBi/giphy.gif)

Reply:
Do you know where these changes are coming from?
I'll open an issue on their bug tracker, and we can close the discussion here.

Reply:
https://github.com/weaveworks/kured/commit/16109017cec02a8a0c4f80f829acc38ab28a14b2

Reply:
Ref: https://github.com/weaveworks/kured/issues/234